tools/project_manager/diff.php

```diff
@@ -7,6 +7,7 @@
 include_once($relPath.'links.inc');
 include_once($relPath.'misc.inc'); // get_integer_param(), attr_safe()
 include_once($relPath."DifferenceEngineWrapper.inc");
+include_once($relPath."DifferenceEngineWrapperBB.inc");
```

Comment:
Your PR is missing at least the `DifferenceEngineWrapperBB.inc` file 😁

Reply:
Oops - thanks.

Reply:
By the way, the `DiffFormatter` include file in `pinc/3rdparty/mediawiki` has a `.php` extension, so originally, I gave my `BBDiffFormatter` which extends the base class a `.php` extension too. One of the security checks then failed, and I guessed it was because `.php` files are checked differently to `.inc`. I therefore renamed my include file to `BBDiffFormatter.inc`. Hope that's OK - if not, what do I need to do to make it pass the security test if it's an include file but with a `.php` extension.